config/dexs.json

```
@@ -919,6 +919,19 @@
     "0xAC4c6e212A361c968F1725b4d055b47E63F80b75",
     "0xd30d8ca2e7715ee6804a287eb86fafc0839b1380"
   ],
+  "ronin": [
```

Biome (2.1.2)

[error] The key ronin was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)